src/plugins/intel_cpu/src/emitters/snippets/jit_snippets_call_args.hpp

```diff
@@ -16,7 +16,9 @@ namespace ov::intel_cpu {
 #if defined(OPENVINO_ARCH_ARM64)
 #    define SNIPPETS_MAX_DATA_PTR_COUNT 23
 #else
-#    define SNIPPETS_MAX_DATA_PTR_COUNT 11
+// Issue: 168073
+// TODO: decrease max array size
+#    define SNIPPETS_MAX_DATA_PTR_COUNT 24
```

Comment:
I meant to replace value with 24 only for only `buffer_offset` array. `src_ptrs` and `dst_ptrs` should have 11 for now because it's maximum possible count of available GPRs. We cannot have inputs and outputs greater than 11 in total. But we can have more 11 buffer clusters which define buffer offsets here.

So please revert this change and replace `SNIPPETS_MAX_DATA_PTR_COUNT` with 24 only on L50.

Reply:
Done